CHANGELOG.md

```diff
@@ -35,34 +35,49 @@ Markdown Spec](https://github.github.com/gfm/).
 - Removed various invalid / unnecessary ARIA role specifications
 - Fixed poor alternative text on some thumbnails
 - Fixed accessibility issues with the global search form
-- Logging now works as expected
+- Logging to `log/debug.log` file in addition to the console
 - Pinned docker-compose users to RAIS 3.x to avoid breakages when 4.x ships
 
 ### Added
- - Enabled apache mod_ssl in web image build
- - Included codebase as part of Dockerfile build
+- Enabled apache mod_ssl in web image build
+- Included codebase as part of Dockerfile build
+- `ONI_STORAGE_PATH` env var, missed in previous edits to `docker-compose.yml`
```

Comment:
This is something that may be useful for bare-metal installs, but we *don't* want people to have the option to change it in the docker-compose setup unless they **really** know what they're doing.  Without properly changing mount points, altering `ONI_STORAGE_PATH` will make ONI stop functioning.

Reply:
Removed. I discussed this and the broader concepts with Andrew and I think he gave me a better grasp of why this is bad practice in general for Docker. I have other ideas for cleaning up things with Docker env vars and some of our bash scripting, but I'll make a separate PR for those to not hold this up :+1: